lib/ordered_map.h

```diff
@@ -59,7 +59,7 @@ class ordered_map {
     struct mapcmp : std::binary_function<const K*, const K*, bool> {
         COMP    comp;
         bool operator()(const K *a, const K *b) const { return comp(*a, *b); } };
-    typedef std::map<const K *, iterator, mapcmp, ALLOC>  map_type;
+    typedef std::map<const K *, iterator, mapcmp>  map_type;
```

Comment:
This avoids the problem by using the default allocator, which is wrong if anyone ever tries to use any other allocator...